x-pack/platform/plugins/shared/onechat/public/components/nav_control/onechat_nav_control.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import type { OnechatPluginStart } from '../../types';
+
+interface OnechatNavControlServices {
+  onechat: OnechatPluginStart;
+}
+
+const iconType: React.FC<SVGProps<SVGSVGElement>> = (props) => (
+  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16" {...props}>
+    <path d="M8.14062 10.1279C8.33131 9.93073 8.64571 9.92389 8.84473 10.1133C9.04442 10.3037 9.05251 10.6204 8.8623 10.8203L8.5 10.4756L8.86133 10.8213L8.86035 10.8223L8.8584 10.8232L8.85449 10.8281C8.85175 10.8309 8.8484 10.8343 8.84473 10.8379C8.83688 10.8456 8.82656 10.8551 8.81445 10.8662C8.79019 10.8885 8.7574 10.9175 8.71582 10.9502C8.63226 11.0159 8.51258 11.0993 8.35742 11.1807C8.04514 11.3445 7.59082 11.5 7 11.5C6.40918 11.5 5.95486 11.3445 5.64258 11.1807C5.48742 11.0993 5.36774 11.0159 5.28418 10.9502C5.2426 10.9175 5.20981 10.8885 5.18555 10.8662C5.17345 10.8551 5.16312 10.8456 5.15527 10.8379C5.1516 10.8343 5.14825 10.8309 5.14551 10.8281L5.1416 10.8232L5.13965 10.8223L5.1377 10.8203C4.94749 10.6204 4.95558 10.3037 5.15527 10.1133C5.35429 9.92389 5.66869 9.93073 5.85938 10.1279L5.86133 10.1299C5.86828 10.1363 5.88278 10.1489 5.90332 10.165C5.94479 10.1976 6.01311 10.2454 6.10742 10.2949C6.29514 10.3934 6.59097 10.5 7 10.5C7.40903 10.5 7.70486 10.3934 7.89258 10.2949C7.98689 10.2454 8.05521 10.1976 8.09668 10.165C8.11722 10.1489 8.13172 10.1363 8.13867 10.1299L8.14062 10.1279Z" />
+    <path d="M5 7C5.55228 7 6 7.44772 6 8C6 8.55229 5.55228 9 5 9C4.44772 9 4 8.55229 4 8C4 7.44772 4.44772 7 5 7Z" />
+    <path d="M9 7C9.55229 7 10 7.44772 10 8C10 8.55229 9.55229 9 9 9C8.44771 9 8 8.55229 8 8C8 7.44772 8.44771 7 9 7Z" />
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M7 0C7.82843 0 8.5 0.671573 8.5 1.5C8.5 2.15281 8.08218 2.70597 7.5 2.91211V4H9C10.6569 4 12 5.34315 12 7H13L13.1025 7.00488C13.6067 7.05621 14 7.48232 14 8V11C14 11.5523 13.5523 12 13 12H12V13C12 13.5523 11.5523 14 11 14H3C2.44772 14 2 13.5523 2 13V12H1C0.447715 12 0 11.5523 0 11V8C0 7.44772 0.447715 7 1 7H2C2 5.34315 3.34315 4 5 4H6.5V2.91211C5.91782 2.70597 5.5 2.15281 5.5 1.5C5.5 0.671573 6.17157 0 7 0ZM5 5C3.89543 5 3 5.89543 3 7V13H11V7C11 5.96435 10.2128 5.113 9.2041 5.01074L9 5H5ZM1 11H2V8H1V11ZM12 11H13V8H12V11ZM7 1C6.72386 1 6.5 1.22386 6.5 1.5C6.5 1.77614 6.72386 2 7 2C7.27614 2 7.5 1.77614 7.5 1.5C7.5 1.22386 7.27614 1 7 1Z"
+    />
+  </svg>
+);
+
+const RobotIcon = ({ size = 'm', ...rest }: Omit<EuiIconProps, 'type'>) => {
+  return <EuiIcon type={iconType} size={size} {...rest} />;
+};
+
+const buttonCss = css`
+  padding: 0px 8px;
+`;
+
+export function OnechatNavControl() {
+  const {
+    services: { onechat },
+  } = useKibana<OnechatNavControlServices>();
+
+  return (
+    <EuiToolTip content={buttonLabel}>
+      <EuiButton
+        css={buttonCss}
+        aria-label={buttonLabel}
+        data-test-subj="OnechatNavControlButton"
+        onClick={() => {
+          onechat.openConversationFlyout({});
```

Comment:
options should not be provided, so it defaults to options provided by apps thru setConversationFlyoutActiveConfig